### DISCUSSION TITLE
C++ Implementation

### DISCUSSION THREAD

**Discussion by senceryazici** [anatolianroverchallenge/rscp#13] [Ideas]
Hello,

There will be the addition of C++ library for this protocol in the very near future. Drop a like if you're interested.

**Comment by MiroslawRajczyk** [comment]
Hi, how's it going with the C++ library that you promised? One of the organizers during day 0 said, that the C++ library is ready on the repository. 

Best Regards

**Reply by senceryazici** [reply]
Hi,

Because there was very little interest to the topic, this **optional** addition will not be worked on. Thus closing. Best!

**Comment by senceryazici** [comment]
Incoming multi-language support [here](https://github.com/anatolianroverchallenge/rscp/discussions/17)! Brace yourselves!!
